src/profiler/symbolinfo.cpp

```diff
@@ -158,9 +158,9 @@ void SymbolInfo::loadSymbolsUsing(DbgHelp* dbgHelp, const std::wstring& sympath)
 			// This is a secondary dbgHelp, so just complement debug
 			// information for modules that have none.
 			
-			for (size_t n=0;n<modules.size();n++)
+			for (size_t m=0;m<modules.size();m++)
 			{
-				Module &mod = modules[n];
+				Module &mod = modules[m];

```

Comment:
It might be too late for that, but generally it's better to place unrelated changes in separate commits. That would also provide for a place to describe why the change is done (i.e. which compiler warning is fixed if any), in the commit message.

Reply:
I assumed you'd prefer a single commit. I squashed what few separate commits I had into one before pushing, sorry about that.
The local variable name changes here and there are to fix class / outer scope variables being shadowed.

Reply:
Some good advice can be found by asking the right question:

https://www.google.com/search?q=when+to+commit+git

For instance:

> You shouldn't commit based on a time basis, but on a feature basis. Whenever you add a new feature that's worth committing, commit. You added a working method? Commit. You fixed a typo? Commit. You fixed a file's wrong indentation? Commit. There's nothing wrong committing small changes, as soon as the commit is relevant.
> 
> What is wrong is committing a huge number of changes, with no relations between each others. It makes it very hard to detect the commit source of a given regression.


-- https://softwareengineering.stackexchange.com/a/74893/16481